### PR TITLE
[GEN-1239] Update data guide for 16.0-public

### PR DESCRIPTION
**Pull request by Chelsea-Na**
- Added the crosswalk table between files
- Added general updates to the field and file descriptions
- Re-arranged some of the sections for clarity
- Added minor edits for clarity

### CODE REVIEW

File: scripts/data_guide/data_guide.qmd [hunk 1]
```diff
@@ -269,7 +269,7 @@ Samples that have variants that could be merged together will be filtered out. T
 
 [Genome Nexus Annotation Status]{.underline}
 
-Variants that are unable to be annotated by Genome Nexus are removed.
+Mutation data that fail annotation by Genome-Nexus are excluded from the release. For example, this occurs when the allele extracted from the VCF or MAF file does not match the reference allele, or when the VEP tool is unable to annotate the data.
```
Comment (BryanFauble) [inline scripts/data_guide/data_guide.qmd:272]:
```suggestion
Mutation data that fails to be annotation by Genome-Nexus is excluded from the release. For example, this occurs when the allele extracted from the VCF or MAF file does not match the reference allele, or when the VEP tool is unable to annotate the data.
```

File: scripts/data_guide/data_guide.qmd [hunk 2]
```diff
@@ -431,194 +432,227 @@ for (center in sort(processed_centers)) {
...
-
-`AGE_AT_SEQ_REPORT_DAYS` [Not available for public releases]
-
-* Expected Values: [Integer], >32485, <6570, Unknown
-* Data Description: The interval in days between the patient's date of birth and the date of the sequencing report that is associated with the sample.  The interval is masked for >32485 and <6570.
+* Data Description: The year the sample was sequenced.
+
+### Linking clinical data to genomic data
+
++---------------------------+-----------------------------+----------------------+-----------------------+
```
Comment (BryanFauble) [inline scripts/data_guide/data_guide.qmd:643]: I personally use https://marketplace.visualstudio.com/items?itemName=darkriszty.markdown-table-prettify to re-format markdown tables. It shouldn't cause problems not being formatted, but it's a bit nicer when looking at the non-rendered content.

**Review by BryanFauble — APPROVED** [review]
Great changes here!

File: scripts/data_guide/data_guide.qmd [hunk 3]
```diff
@@ -134,7 +134,7 @@ AACR Project GENIE data versions follow a numbering scheme derived from [semanti
 
 AACR Project GENIE Data is currently available via two mechanisms:
 
--   Synapse Platform (Sage Bionetworks): https://synapse.org/genie
+-   Synapse Platform (Sage Bionetworks): https://genie.synapse.org/ or https://synapse.org/genie
```
Comment (thomasyu888) [inline scripts/data_guide/data_guide.qmd:137]: Should we just link to the portal?

File: scripts/data_guide/data_guide.qmd [hunk 4]
```diff
@@ -230,16 +230,16 @@ Any sample with `SAMPLE_CLASS` that is cfDNA is removed from the public release.
...
 
 [Age information redaction]{.underline}
 
-When `AGE_AT_SEQ_REPORT`, `INT_CONTACT`, or `INT_DOD` is >32485 or <6570, the fields `BIRTH_YEAR`, `YEAR_CONTACT`, `YEAR_DEATH`, `INT_CONTACT`, `INT_DOD`, and `AGE_AT_SEQ_REPORT` are redacted. If the difference between the `BIRTH_YEAR` and `YEAR_CONTACT` or `BIRTH_YEAR` and `YEAR_DEATH` is greater than 89, those values will be redacted.
+When `AGE_AT_SEQ_REPORT`, `INT_CONTACT`, or `INT_DOD` is \>32485 or <6570 days, the fields `BIRTH_YEAR`, `YEAR_CONTACT`, `YEAR_DEATH`, `INT_CONTACT`, `INT_DOD`, and `AGE_AT_SEQ_REPORT` are redacted. If the difference between the `BIRTH_YEAR` and `YEAR_CONTACT` or `BIRTH_YEAR` and `YEAR_DEATH` is greater than 89 years, those values will be redacted.
 
--   Intervals are redacted with ">32485" and "<6570"
--   Ages are redacted with ">89" and "<18"
--   Years are redacted with "cannotReleaseHIPAA" and "withheld".
+-   Intervals are redacted with "\>32485" and "<6570"
```
Comment (thomasyu888) [inline scripts/data_guide/data_guide.qmd:240]: Why the `\`?

File: scripts/data_guide/data_guide.qmd (hunk 1 above)
Comment (thomasyu888) [inline scripts/data_guide/data_guide.qmd:272]: Is it worth while to point out genome nexus calls VEP, or link to genome nexus as a tool? Or do we do that earlier?

File: scripts/data_guide/data_guide.qmd [hunk 5]
```diff
@@ -280,9 +280,9 @@ Variants that are unable to be annotated by Genome Nexus are removed.
 
 [Gene symbol harmonization]{.underline}
 
-All submitted HUGO gene symbols will be harmonized against GRCh37 by checking the coordinates against the annotation. The below diagram illustrates how conflicts between submitted and harmonized symbols will be resolved.
+All submitted HUGO gene symbols will be harmonized against GRCh37 by checking the coordinates against the annotation.
 
--   Every single row of your bed file will be matched against the gene database. If the submitted symbol matches a row in the gene database and there is an overlap from the submitted bed region, the submitted symbol will be returned.
+-   Every single row of the bed file will be matched against the gene database. If the submitted symbol matches a row in the gene database and there is an overlap from the submitted bed region, the submitted symbol will be returned.
```
Comment (thomasyu888) [inline scripts/data_guide/data_guide.qmd:285]: Do we need to say how we build the gene database?

**Review by thomasyu888 — APPROVED** [review]
🔥 thanks for the review Bryan and great work here Chelsea.  See my comments but I'm going to pre-approve!

File: scripts/data_guide/data_guide.qmd (hunk 3 above)
Comment (Chelsea-Na) [reply scripts/data_guide/data_guide.qmd:137]: I think this is fine for now. Will probably want to do this next data guide update!

File: scripts/data_guide/data_guide.qmd (hunk 1 above)
Comment (Chelsea-Na) [reply scripts/data_guide/data_guide.qmd:272]: A combination probably works: `Mutation data that fails to be annotated by Genome-Nexus is excluded from the release.`

Comment (Chelsea-Na) [reply scripts/data_guide/data_guide.qmd:272]: Will add this correction and add a link to the tool in the next commit.

File: scripts/data_guide/data_guide.qmd (hunk 5 above)
Comment (Chelsea-Na) [reply scripts/data_guide/data_guide.qmd:285]: How do we build the gene database?

Comment (thomasyu888) [reply scripts/data_guide/data_guide.qmd:285]: The database was created here: http://ftp.ensembl.org/pub/release-75/gtf/homo_sapiens/Homo_sapiens.GRCh37.75.gtf.gz

**Review by thomasyu888 — APPROVED** [review]
🔥 LGTM!

**Pull request merged**